v2/send.go

```diff
@@ -13,6 +13,12 @@ type SendBody struct {
 	Brand    string      `json:"brand,omitempty"`
 }
 
+// SendMessageRequestBody is the JSON body expected by
+// Courier's /send endpoint that accepts a message object
+type SendMessageRequestBody struct {
+	Message interface{} `json:"message"`
```

Comment:
is this a generic placeholder? I don't read Go that well so it is a naive question

Reply:
Go typing is shit. no way to do complex nested, unions etc. So best way to keep things going is interface{} at top level which can take in anything

Reply:
so to ans your ques yep a generic placeholder